static/js/components/dashboard/CourseDescription_test.js

```diff
@@ -31,80 +35,102 @@ export function findCourse(courseSelector: (course: Course, program: Program) =>
 }
 
 describe('CourseDescription', () => {
-  const now = moment();
+  let createCourseRun = (status: string, courseEndDate: string, courseStartDate: string) => {
+    return Object.assign({
+      "position": 1,
+      "title": "title",
+      "course_id": "course-v1:odl+GIO101+FALL14",
+      "status": status,
+      "id": 1,
+      "fuzzy_start_date": "Fall 2017",
+      "course_end_date": courseEndDate,
+      "course_start_date": courseStartDate
+    });
+  };
 
   it('shows the course title', () => {
     for (let status of ALL_COURSE_STATUSES) {
       let course = findCourse(course => course.status === status);
-      const wrapper = shallow(<CourseDescription course={course} now={now}/>);
+      const wrapper = shallow(<CourseDescription course={course}/>);
+
       assert(course.title.length > 0);
       assert.equal(wrapper.find(".course-description-title").text(), course.title);
     }
   });
 
-  it('shows (enrolled) only for enrolled courses', () => {
-    for (let status of ALL_COURSE_STATUSES) {
-      let enrolledCourse = findCourse(course => course.status === status);
-      const wrapper = shallow(<CourseDescription course={enrolledCourse} now={now} />);
-      let text = wrapper.find(".course-description-enrolled").text();
-      if (status === STATUS_ENROLLED_NOT_VERIFIED) {
-        assert.equal(text, "(enrolled)");
-      } else {
-        assert.equal(text, "");
-      }
-    }
-  });
+  it(`does show date with status passed`, () => {
+    let course = _.cloneDeep(findCourse(course => course.status === STATUS_PASSED));
+    course.runs = [
+      createCourseRun(STATUS_PASSED, '2016-10-09T10:20:10Z', '2016-10-01T10:20:10Z')
```

Comment:
a few things: 

1. do these `courseStartDate` and `courseEndDate` arguments ever change in the 6 times you call this function? it looks like the same values are repeated many times. it would help to have default values for these arguments. that way we can easily identify which test cases use a 'special' date range
1. are these values sensitive to the actual current datetime? in other words, will anything change when the current datetime is inside this range, or beyond this range? just making sure